docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -82,6 +80,10 @@ The preceding example also shows the use of `_` as a *digit separator*, which is
 The type of an integer literal is determined by its suffix as follows:
 
 - If the literal has no suffix, its type is the first of the following types in which its value can be represented: `int`, `uint`, `long`, `ulong`.
+
+  > [!NOTE]
+  > Literals are interpreted as positive values. For example, the literal `0xFF_FF_FF_FF` represents the number `4294967295` of the `uint` type, though it has the same bit representation as the number `-1` of the `int` type. If you need a value of a certain type, cast a literal to that type. Use the `unchecked` operator, if a literal value cannot be represented in the target type. For example, `unchecked((int)0xFF_FF_FF_FF)` produces `-1`.
```

Comment:
This is good, although perhaps to be perfectly accurate this should say "Literals are interpreted as positive values, unless preceded by a minus sign." ?

Reply:
To be perfectly accurate, a minus sign is not the part of a literal. It's the unary minus operator. So, first, a literal is interpreted and then the minus operator is applied to that value. (There are two exceptions to that logic, but it's really the spec material.)

Reply:
Ok, I see, thanks!